src/main/java/br/com/caelum/clines/api/aircraftmodels/AircraftModelForm.java

```diff
@@ -0,0 +1,18 @@
+package br.com.caelum.clines.api.aircraftmodels;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
```

Comment:
Given that you have restricted the level of access in the constructor to the package

It makes sense to maintain the class's level of access as public?

Reply:
Any sugestion?

Reply:
Remove this restriction or apply this restriction to whole class in this package